### PR TITLE
Test Relocatable AddUint

### LINKED ISSUE #120: Test addition between `Relocatable` and `uint`
Add a test where given a `Relocatable` and a `uint` the test expects the return value to be a new `Relocatable` with its `SegmentIndex` and `Offset` fields computed correctly.

The method is defined [here](https://github.com/lambdaclass/cairo-vm.go/blob/c388ec512bfc1e073eb6bbb007f91ea51933c0cd/pkg/vm/memory/relocatable.go#L88-L91).

### PR DESCRIPTION
**Pull request by mattcoburn24**
closes #120 
added TestRelocatableAddUint

### CODE REVIEW

**Comment by mattcoburn24** [comment]
@gabrielbosio in #120, the method you linked shows an error being returned from Relocatable.AddFelt but looks like it was removed from main so i didn't check for it in my PR, Was it supposed to be removed from main?

File: pkg/vm/memory/relocatable_test.go [hunk 1]
```diff
@@ -43,6 +43,15 @@ func TestMaybeRelocatableAddFelt(t *testing.T) {
 	}
 }
 
+func TestRelocatableAddUint(t *testing.T) {
+	rel := memory.Relocatable{2, 4}
+	res := rel.AddUint(24)
+	expected := memory.Relocatable{2, 28}
+	if res != expected {
+		t.Fatalf("got wrong value from Relocatable.AddUint, expected: %v, got: %v", expected, res)
```
Comment (toni-calvin) [inline pkg/vm/memory/relocatable_test.go:51]:
```suggestion
		t.Errorf("got wrong value from Relocatable.AddUint, expected: %v, got: %v", expected, res)
```

Why have you decided to use Errorf?

**Comment by gabrielbosio** [comment]
@coburn24 yes, that was the idea. I updated the description of #120, thanks!

File: pkg/vm/memory/relocatable_test.go (hunk 1 above)
Comment (gabrielbosio) [inline pkg/vm/memory/relocatable_test.go]: Why does this test uses `Fatalf` and not `Errorf`?

Comment (mattcoburn24) [reply pkg/vm/memory/relocatable_test.go]: Oops, updated to use Errorf 👍🏼 
@gabrielbosio

**Review by jrchatruc — APPROVED** [review]

**Review by gabrielbosio — APPROVED** [review]

**Pull request merged**